src/main/java/com/hierynomus/smbj/utils/SmbFileAttributeUtils.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C)2016 - SMBJ Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hierynomus.smbj.utils;
+
+/**
+ * This utility class provides utilities for file attributes.
+ *
+ */
+public class SmbFileAttributeUtils {
```

Comment:
Could you maybe submit this as a separate PR to `SmbFiles` this is really useful and I missing such a good way to check if its a file or directory!